src/data/roadmaps/postgresql-dba/postgresql-dba.json

```
@@ -578,15 +578,15 @@
     },
     {
       "id": "2zqZkyVgigifcRS1H7F_b",
-      "type": "button",
+      "type": "DevOps Roadmap",
```

```suggestion
      "type": "button",
```

I think it will button not DevOps Roadmap